### PR TITLE
mbuffer: Clarify gpl3 license

### PR DESCRIPTION
**Pull request by tokudan**
###### Motivation for this change

Clarification of the license

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/mbuffer/default.nix [hunk 1]
```diff
@@ -25,7 +25,7 @@ stdenv.mkDerivation rec {
   meta = {
     homepage = "http://www.maier-komor.de/mbuffer.html";
     description  = "A tool for buffering data streams with a large set of unique features";
-    license = lib.licenses.gpl3;
+    license = lib.licenses.gpl3Only;
```
Comment (fabaff) [inline pkgs/tools/misc/mbuffer/default.nix:28]: The file headers say `GPLv3+`.

```suggestion
    license = lib.licenses.gpl3Plus;
```

**Review by fabaff — CHANGES_REQUESTED** [review]

File: pkgs/tools/misc/mbuffer/default.nix (hunk 1 above)
Comment (tokudan) [reply pkgs/tools/misc/mbuffer/default.nix:28]: Nice catch, though the README states gpl3Only:
```
LICENSE:
========
GNU GPLv3 (see file LICENSE for details)
```
The LICENSE file is just a copy of GPL3.
I'll try to contact the author to clarify this.

File: pkgs/tools/misc/mbuffer/default.nix [hunk 2]
```diff
@@ -25,7 +25,7 @@ stdenv.mkDerivation rec {
   meta = {
     homepage = "http://www.maier-komor.de/mbuffer.html";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/mbuffer/default.nix:26]:
```suggestion
    homepage = "https://www.maier-komor.de/mbuffer.html";
```

File: pkgs/tools/misc/mbuffer/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/tools/misc/mbuffer/default.nix:28]: The license file is the same for both variants. Later also include a ``or at your option any later version`` in the source files.

Comment (tokudan) [reply pkgs/tools/misc/mbuffer/default.nix:28]: The response from Thomas is that he wants to check newer versions before accepting them, so gpl3Only is the correct choice here.

**Pull request merged**